### ISSUE TITLE
Blogpost validation

### ISSUE THREAD

**Issue by DavidBruant** [dtc-innovation/dtc-innovation.github.io#13]
It's been a few days https://github.com/dtc-innovation/dtc-innovation.github.io/pull/10 is in queue with no review (we briefly talked about it with @ClementineHahn ).

What about a 2-phase review system ?
1. does the blogpost threaten dtc (image, reputation, financially, values, etc.)? If no, publish
2. send PRs as things go to change/fix the content

**Comment by thom4parisot** [comment]
I skimmed though it and totally forgot to go back to it later, my bad.

If it's done in 2 phases as described above, it kind of defeats the purpose of the review – which is to add value to what is brought through the writing.

I feel more comfortable with the idea of _publishing it together_ rather than having to fix stuff in the rush because the blog post has been sent off the fence.

That being said, the exact same situation happened when I assembled a few pieces about the website. Some nods were sent but after publication… But again, I don't think the 2-phases review system brings us into doing authoring together. It makes the publishing phase going faster for sure. But are we talking about speed or about collaboration?
